### PR TITLE
feat: update UI of interface settings

### PR DESCRIPTION
**Pull request by wheatjs**
Updated interface for the font selector and a preview post in the interface section

image

### CODE REVIEW

File: pages/settings/interface/index.vue [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
 const { t } = useI18n()
 
 useHeadFixed({
   title: () => `${t('settings.interface.label')} | ${t('nav.settings')}`,
 })
+
+const loremStatus = computed(() => {
+  if (currentUser.value) {
+    const loremStatus: mastodon.v1.Status = {
+      account: currentUser.value!.account,
```
Comment (sxzz) [inline pages/settings/interface/index.vue:12]: maybe we should also have a fake user when there's no user signed in.

File: pages/settings/interface/index.vue [hunk 2]
```diff
@@ -14,6 +41,10 @@ useHeadFixed({
       </div>
     </template>
     <div p6 flex="~ col gap6">
+      <div v-if="loremStatus" border="~ base rounded" bg-active>
+        <StatusCard :status="loremStatus" />
```
Comment (sxzz) [inline pages/settings/interface/index.vue:45]: action buttons should be disabled or hidden.

File: components/settings/SettingsFontSize.vue [hunk 3]
```diff
@@ -4,12 +4,61 @@ import type { FontSize } from '~/types'
 
 const sizes = ['xs', 'sm', 'md', 'lg', 'xl'] as FontSize[]
 const fontSize = useFontSizeRef()
+
+const setFontSize = (e: any) => {
+  if ('valueAsNumber' in e.target)
+    fontSize.value = sizes[e.target.valueAsNumber]
+}
```
Comment (sxzz) [inline components/settings/SettingsFontSize.vue:11]:
```suggestion
const setFontSize = (e: Event) => {
  if (e.target && 'valueAsNumber' in e.target)
    fontSize.value = sizes[e.target.valueAsNumber as number]
}
```

File: pages/settings/interface/index.vue (hunk 1 above)
Comment (wheatjs) [reply pages/settings/interface/index.vue:12]: Agreed that would be better, should we use the Elk account details as the fake user?

File: pages/settings/interface/index.vue (hunk 2 above)
Comment (wheatjs) [reply pages/settings/interface/index.vue:45]: Perhaps we could add a disabled property to the status card that would disable interactions with it?

Comment (sxzz) [reply pages/settings/interface/index.vue:45]: Sure!

File: pages/settings/interface/index.vue (hunk 1 above)
Comment (sxzz) [reply pages/settings/interface/index.vue:12]: Yes!

**Comment by edimitchel** [comment]
What about don't let the slider component be affected by the font size change until the mouse is release?
Because it makes very difficult to change the value properly.
Also, I would suggest to put the slider on top of the setting page to don't be disturbed

**Comment by wheatjs** [comment]
> What about don't let the slider component be affected by the font size change until the mouse is release?
> Because it makes very difficult to change the value properly.
> Also, I would suggest to put the slider on top of the setting page to don't be disturbed 

I agree with the first point. As for the second I'm not sure about putting it above the preview as it may look weird and cause a strange break in content.

**Comment by wheatjs** [comment]
Going to close as I don't think brings much of an improvement to the UX of the app

**Pull request closed without merging**